packages/quote-block/src/QuoteBlock.tsx

```diff
@@ -1,26 +1,31 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
 import { useBlockSettings, useEditorState } from '@frontify/app-bridge';
-import { RichTextEditor } from '@frontify/fondue';
+import { EditorActions, RichTextEditor } from '@frontify/fondue';
 import '@frontify/fondue-tokens/styles';
 import { toRgbaString, useGuidelineDesignTokens } from '@frontify/guideline-blocks-shared';
 import { FC } from 'react';
 import 'tailwindcss/tailwind.css';
-import { DEFAULT_AUTHOR_NAME, DEFAULT_COLOR_VALUE } from './settings';
+import { DEFAULT_COLOR_VALUE } from './settings';
 import {
     ContentWithAuthorProps,
     LineType,
-    LineWidth,
     Props,
     QuoteSize,
     QuoteStyle,
     QuoteType,
     Settings,
-    lineWidthMap,
     quoteSizeMap,
 } from './types';
 import { quoteIconMap } from './utilities';
 
+const PLACEHOLDER = 'Add your quote text here';
```

Comment:
You could remove the variable assignment and just add the string directly to the `RichTextEditor` component, as it is not used more than once.